assets/src/blocks/Cookies/CookiesBlock.js

```diff
@@ -4,9 +4,11 @@ export const BLOCK_NAME = 'planet4-blocks/cookies';
 
 export const registerCookiesBlock = () => {
   const {registerBlockType} = wp.blocks;
+  const {__} = wp.i18n;
 
   registerBlockType(BLOCK_NAME, {
     title: 'Cookies',
+    description: __('Display real time signatures being collected worldwide on petitions.', 'planet4-blocks-backend'),
```

Comment:
I think this isn't the correct description for this block? 😛

Reply:
Fixed!